src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/StorageContainerTask.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.models.BlobContainerItem;
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.ContainerProcessor;
+import uk.gov.hmcts.reform.blobrouter.util.StorageClientManager;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class StorageContainerTask {
+
+    private static final Logger LOGGER = getLogger(StorageContainerTask.class);
+
+    private final BlobServiceAsyncClient storageClient;
+    private final ContainerProcessor containerProcessor;
+    private final ServiceConfiguration serviceConfiguration;
+
+    public StorageContainerTask(
+        BlobServiceAsyncClient storageClient,
+        ContainerProcessor containerProcessor,
+        ServiceConfiguration serviceConfiguration
+    ) {
+        this.containerProcessor = containerProcessor;
+        this.serviceConfiguration = serviceConfiguration;
+        this.storageClient = storageClient;
+    }
+
+    public void run() {
+        LOGGER.info("Started job");
```

Comment:
I know that we'll have logger name, but it would still be worth mentioning what kind of job we started (and finished).